setup.py

```diff
@@ -0,0 +1,20 @@
+from distutils.core import setup
+
+setup(
+    name='GPS Renda',
```

Comment:
in my mind I think I have just been calling this package `gpsrenda`

Reply:
Yes I think `name` corresponds to `pip install {name}` if you register the package. I thought it was a "colloquial name".